### DISCUSSION TITLE
Welcome to Skin-Cancer-Classification-using-Deep-Learning Discussions!

### DISCUSSION THREAD

**Discussion by Tirth27** [Tirth27/Skin-Cancer-Classification-using-Deep-Learning#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hello!

**Comment by avinashkolla532** [comment]
![Screenshot 2023-04-17 143419](https://user-images.githubusercontent.com/108348436/232453130-2bd2584b-f099-4786-aa6b-a88921a5bbda.png)

![Screenshot 2023-04-17 143419](https://user-images.githubusercontent.com/108348436/232453024-2e278019-c90d-4113-a862-18433a4b2606.png)
 I am getting like this while doing ng serve
